### PR TITLE
mpvScripts.mpv-playlistmanager: init at unstable-2021-03-09

### PR DESCRIPTION
**Pull request by lunik1**
###### Motivation for this change

mpv lua scripts that allows manipulation of playlists

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 119898` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mpvScripts.mpv-playlistmanager</li>
  </ul>
</details>

**Review by legendofmiracles — CHANGES_REQUESTED** [review]
use stdenvNoCC instead of stdenv

File: pkgs/applications/video/mpv/scripts/mpv-playlistmanager.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub, youtube-dl }:
```
Comment (legendofmiracles) [inline pkgs/applications/video/mpv/scripts/mpv-playlistmanager.nix:1]:
```suggestion 
{ lib, stdenvNoCC, fetchFromGitHub, youtube-dl }:

stdenvNoCC.mkDerivation rec {
```

**Comment by lunik1** [comment]
I think most of the other mpv scripts to not need a C compiler but use `stdenv`; worth patching in another PR?

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 119898` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mpvScripts.mpv-playlistmanager</li>
  </ul>
</details>

**Comment by legendofmiracles** [comment]
If all still build, why not. But someone else should check if that would work

**Comment by legendofmiracles** [comment]
I just tested it, all but one do not need a C compiler. 
Maybe for the sake of all all mpv plugins being as close to each other as possible it should be kept. I don't know honestly

**Comment by lunik1** [comment]
Perhaps this should be put to mpv/mpv plugin maintainers?

File: pkgs/applications/video/mpv/scripts/mpv-playlistmanager.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{ lib, stdenvNoCC, fetchFromGitHub, youtube-dl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mpv-playlistmanager";
+  version = "unstable-09-03-2021";
```
Comment (SuperSandro2000) [inline pkgs/applications/video/mpv/scripts/mpv-playlistmanager.nix:5]:
```suggestion
  version = "unstable-2021-03-09";
```

**Comment by SuperSandro2000** [comment]
> I think most of the other mpv scripts to not need a C compiler but use `stdenv`; worth patching in another PR?

Sure, go ahead.

> Perhaps this should be put to mpv/mpv plugin maintainers?

There is no such group. Just change it and if all plugins still work we can merge it.

**Comment by lunik1** [comment]
I have done this at #119968 along with a couple of other cleanup items

**Pull request merged**